src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -426,6 +428,12 @@ export const FallbackAuthEntry = React.createClass({
         }
     },
 
+    focus: function() {
```

Comment:
I'm not sure what would call this, could you clarify?

Reply:
Please see the comment at line 64. Apparently InteractiveAuthEntryComponents should implement that if they need it. I've got an inspiration from PasswordAuthEntry component otherwise I would just do this inside componentDidMount function.

Reply:
Hmm. Okay, I've managed to find where this is actually called. We're doing fun things in InteractiveAuth to call it.